src/components/Careers/InterviewProcess/index.tsx

```diff
@@ -51,9 +51,9 @@ export const InterviewProcess = () => {
         <div className="pt-24 mb-16 text-center" id="interview-process">
             <Structure.Section width="3xl">
                 <Structure.SectionHeader
-                    title="Interview process"
+                    title="Get paid to try working here"
                     titleTag="h2"
-                    leadText="You can expect a fast, yet thorough process. While it differs by role, we usually follow a structure of the following stages:"
+                    leadText="We do ~2 interviews, then pay you to do some real life, or close to real life, work:"
```

Comment:
In the roles I'm interviewing for, I think it tends to be 3? (Screening by Coua, me, technical with Tim) - but this might be atypical.

Maybe 2-3?